app/models/contact.rb

```diff
@@ -20,6 +20,8 @@ class Contact < ApplicationRecord
   after_create :republish_organisation_to_publishing_api
   after_destroy :republish_organisation_to_publishing_api
 
+  after_commit :republish_embassies_index_page_to_publishing_api
```

Comment:
Just leaving a comment here unrelated to the line but in response to the decisions you've outlined in this commit message that I agree with all the decisions you've made there, thanks for outlining them! 😄

Reply:
Thank you!